content/docs/reference/glossary.md

```diff
@@ -0,0 +1,12 @@
+---
+title: Pulumi Glossary
+menu:
+  reference:
+    weight: 11
+---
+
+This page is a collection of brief definitions of technical terms used in the Pulumi documentation.
+
+{{< glossary >}}
```

Comment:
Do you expect to re-use this shortcode (say, to display portions of the glossary in various places at some pont)? 

If not, I'd probably suggest we use a regular ol' layout instead.